soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -1026,6 +1038,26 @@ void CheckTrackerWindow::DrawElement() {
     }
 }
 
+bool shouldHideArea(ImGuiTextFilter& checkSearch, std::map<RandomizerCheckArea, std::vector<RandomizerCheckObject>> checksByArea, RandomizerCheckArea rcArea) {
+    bool shouldHideFilteredAreas = CVarGetInteger("gCheckTrackerHideFilteredAreas", 1);
```

Comment:
Please rename the CVar to "gTrackers.CheckTracker.HideFilteredAreas". This is in preparation for a CVar rename I have planned, hopefully shortly.